source/administration/migrating.rst

```diff
@@ -190,7 +189,14 @@ Migrating from Slack Using the Mattermost CLI
    
 .. note:: To run the CLI command, you must be in the directory that contains the Mattermost installation. On a default installation of Mattermost, the directory is ``/opt/mattermost/``. Also, if you followed our `installation process <../guides/administrator.html#installing-mattermost>`__, you must run the command as the user *mattermost*. The executable is in the ``bin`` subdirectory and is called ``mattermost``.
 
+Migrating from Slack Using the Mattermost mmetl Tool and Bulk Import
+`````````````````````````````````````````````````````````````````````
+.. note:: This method is the recommended way to import Slack's corporate export file.
 
+1. Use the `mmetl tool <https://github.com/mattermost/mmetl>`_ to transform Slack's corporate export file into the ``jsonl`` format required by the bulk import tool.
```

Comment:
```suggestion
1. Use the `slack-advanced-exporter <https://github.com/grundleborg/slack-advanced-exporter>`_ to download attachments and add user's email addresses to your Slack corporate export file.

2. Use the `mmetl tool <https://github.com/mattermost/mmetl>`_ to transform Slack's corporate export file into the ``jsonl`` format required by the bulk import tool.
```

Added another important step if people want to have the attachments in their exports as this is currently not an inbuilt functionality of mmetl.

Reply:
Agreed. Additional step has been added.